### PR TITLE
SPM-1289: Support filter advisory_type_name[other]

### PR DESCRIPTION
**Pull request by Josca**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: manager/controllers/filter.go [hunk 1]
```diff
@@ -2,12 +2,15 @@ package controllers
...
 	"app/base/database"
+	"app/base/utils"
 	"fmt"
 	"github.com/pkg/errors"
 	"gorm.io/gorm"
 	"strings"
 )
 
+var otherAdvisoryTypes = strings.Split(utils.Getenv("OTHER_ADVISORY_TYPES", "unspecified,unknown"), ",")
+
```
Comment (MichaelMraka) [inline manager/controllers/filter.go:13]: Maybe we could get this list directly from database at startup. E.g.
`select name from advisory_type where name not in ('enhancement', 'bugfix', 'security');`

Comment (Josca) [reply manager/controllers/filter.go]: Ok, updated.

**Review by MichaelMraka — APPROVED** [review]

**Pull request merged**